pkg/rbd/nodeserver.go

```diff
@@ -303,9 +303,25 @@ func (ns *NodeServer) mountVolumeToStagePath(ctx context.Context, req *csi.NodeS
 	// Publish Path
 	fsType := req.GetVolumeCapability().GetMount().GetFsType()
 	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+	existingFormat, err := diskMounter.GetDiskFormat(devicePath)
+	if err != nil {
+		return err
```

Comment:
please log the error message before returning

Reply:
Done.